code/datums/ammo/ammo.dm

```diff
@@ -237,11 +237,18 @@
 
 		P.fire_at(new_target, original_P.firer, original_P.shot_from, P.ammo.max_range, P.ammo.shell_speed, original_P.original) //Fire!
 
-/datum/ammo/proc/drop_flame(turf/T, datum/cause_data/cause_data) // ~Art updated fire 20JAN17
-	if(!istype(T))
+/datum/ammo/proc/drop_flame(turf/Turf, datum/cause_data/cause_data) // ~Art updated fire 20JAN17
+	if(!istype(Turf))
 		return
-	if(locate(/obj/flamer_fire) in T)
+	if(locate(/obj/flamer_fire) in Turf)
 		return
 
-	var/datum/reagent/R = new flamer_reagent_type()
-	new /obj/flamer_fire(T, cause_data, R)
+	var/datum/reagent/reference = GLOB.chemical_reagents_list[flamer_reagent_id] //this is probably a bad way of doing this but it works
+	var/datum/reagent/chemical = new reference.type
+	chemical.intensityfire = reference.intensityfire
+	chemical.durationfire = reference.durationfire
+	chemical.burncolor = reference.burncolor
+	chemical.fire_penetrating = reference.fire_penetrating
+	chemical.burn_sprite = reference.burn_sprite
```

Comment:
```suggestion
	var/datum/reagent/chemical = GLOB.chemical_reagents_list[flamer_reagent_id]
```
/obj/flamer_fire/Initialize is already going to new the chemical anyways and use make_alike() no need to partially do it.

Can confirm this works w/ the nozzle just fine too, but feel free to try it yourself: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/6612aa21-287b-4432-a240-21ec91d4e18c)